src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.logo {
+  display: block;
+}
+
+.nav {
+  &_list ul {
+    display: flex;
+
+    list-style: none;
+    list-style-type: none;
```

Comment:
The properties `list-style: none;` and `list-style-type: none;` are redundant. You can remove one of them to clean up the code without affecting the styling.